domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -1035,7 +1053,11 @@ class ExplorationProgressController @Inject constructor(
     val commandQueue: SendChannel<ControllerMessage<*>>,
     private val installationId: String?,
     private val learnerId: String?,
-    private val learnerAnalyticsLogger: LearnerAnalyticsLogger
+    private val learnerAnalyticsLogger: LearnerAnalyticsLogger,
+    val startSessionTimeMs: Long,
+    var isContinueButtonAnimationSeen: Boolean?,
```

Comment:
It seems like this can no longer be null with the latest code--suggest switching to ``Boolean`` and removing the extra ``let`` below.

Reply:
done